build/bin/build-client-binary

```diff
@@ -24,6 +24,8 @@ gobuild_component() {
     fi
     if [ ${component} = 'pmm-admin' -o ${component} = 'pmm-agent' ]; then
         build_command="cd /tmp/go/src/${extract_path} && make release"
+    elif [ ${component} = 'mongodb_exporter' ]; then
+        build_command="cd /tmp/go/src/${extract_path} && go get -u ./... && make build"
```

Comment:
Nope, we can't use `go get` – that will use the latest commit instead of commit in submodule

Reply:
/cc @EvgeniyPatlan ^

Reply:
@AlekSi Thank you, removed in d310358